scripts/generate.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Do you plan to have the file names with version?

Reply:
Yeah I've not started drafting the release logic yet but I'd assume we would version the bundles yes, I'm not sure that we'd version the output of this script however.